go/src/koding/kites/gateway/gateway.go

```diff
@@ -0,0 +1,315 @@
+package gateway
```

Comment:
first of all i am not good at naming things, what i have seen in wild with naming things "gateway" after, is mostly when they take actions at the backend with the passed request, on other words, they act as an entrance to internal services/networks etc. The functionality for this kite does not look like it :) what about changing it to something like s3keygen?